geonode/base/api/serializers.py

```diff
@@ -787,7 +787,40 @@ class Meta:
 
 
 class SimpleResourceSerializer(DynamicModelSerializer):
+
     class Meta:
         name = "linked_resources"
         model = ResourceBase
         fields = ("pk", "title", "resource_type", "detail_url", "thumbnail_url")
+
+    def to_representation(self, instance: LinkedResource):
+        return {
```

Comment:
Evaluate to use the `super` instead of the plain json:

```python
def to_representation(self, instance: LinkedResource):
        data = super(PersonListSerializer, self).to_representation(instance)
        data.update({"title": f"{'>>> ' if instance.is_target else '<<< '} {instance.title}"})
        return data
```

In this way if we want to define other field we dont have to add them in the "to_representation"

Reply:
Sure, anyway this class is going to be removed as soon as the client is aligned to use the new `linked_to` and `linked_by` elements. This is a quick and dirty implementation.